compiler/src/AsmOutput.sk

```diff
@@ -3723,6 +3732,29 @@ private fun llvmWrite(instr: Stmt, asm: mutable FunAsmDefBuilder): void {
 
     llvmWriteStore(fakeStore, x, asm)
 
+  | ArrayUnsafePtr{id, pos, typ, obj, offset} ->
+    (addr, info, extraBitOffset, addrByteAlignment) = llvmComputeArrayAddress(
+      instr.pos,
+      obj,
+      offset,
+      asm,
+    );
+    // Dummy up a fake Load instruction and let it do the heavy lifting.
+    fakeLoad = Load{
+      id,
+      typ,
+      pos,
+      addr,
+      bitOffset => info.bitOffsets[0] + extraBitOffset,
+      addrByteAlignment,
+      canCSE => optinfo.getInstr(obj).typ.isDeepFrozen(),
+    };
+    llvmWriteGetAddr(
+      fakeLoad,
+      "r" + optinfo.getInstr(id).idValue().toString(),
+      asm,
+    )
```

Comment:
I see. I wonder if it would be easy enough even to just have a function from instr to string that did this string concatenation, to document the connection between this use and those in intrinsics.sk?